ivy/functional/backends/tensorflow/experimental/linear_algebra.py

```diff
@@ -150,6 +152,18 @@ def multi_dot(
     return dot_out
 
 
+def dot(
+    x1: tf.Tensor,
+    x2: tf.Tensor,
+    out: Optional[tf.Tensor] = None
+) -> tf.Tensor:
+    x1, x2 = ivy.promote_types_of_inputs(x1, x2)
+    return tf.experimental.numpy.dot(x1, x2)
```

Comment:
This should probably be [`tensordot()`](https://www.tensorflow.org/api_docs/python/tf/tensordot)

Reply:
I went with the numpy dot function, since tf.tensordot adds overhead in attempting to handle the axes which seems irrelevant here as Ivy already has a full fledged [tensordot function](https://github.com/unifyai/ivy/blob/master/ivy/functional/ivy/linear_algebra.py#L2113).